tests/main/views/test_buyers.py

```diff
@@ -1886,15 +1886,15 @@ def test_show_draft_brief_summary_page(self, framework_status):
             'How to shortlist suppliers',
             'How to evaluate suppliers',
             'How to award a contract',
-            'Download the Digital Outcomes and Specialists contract',
+            'Download the Digital Outcomes and Specialists 4 contract',
```

Comment:
These tests are failing because the `brief_overview.html` template uses `brief.frameworkName` instead of `brief.framework.name`.

Reply:
This test passes on my machine 🤔

Reply:
I'm getting the same errors as Travis when I run locally.

Reply:
(They pass when I change the template to use `framework.name`)

Reply:
The tests pass now that alphagov/digitalmarketplace-test-utils#28 has been merged. I can update the template anyway though.

Reply:
🤔 Aha, running `make test` didn't update the test utils version for me locally, that's why.